src/AppBundle/Controller/CommitteeController.php

```diff
@@ -27,19 +29,24 @@ public function showAction(Committee $committee): Response
     }
 
     /**
-     * @Route("/comites/{uuid}/evenements/ajouter", name="app_committee_add_event")
+     * @Route("/{uuid}/evenements/ajouter", name="app_committee_add_event", requirements={
+     *   "uuid": "%pattern_uuid%"
+     * })
      * @Method("GET|POST")
      * @Security("is_granted('HOST_COMMITTEE', committee)")
      */
     public function addEventAction(Committee $committee): Response
     {
         return $this->render('committee/add_event.html.twig', [
             'committee' => $committee,
+            'committee_hosts' => $this->get('app.committee_manager')->findCommitteeHostsList($committee),
```

Comment:
Yes if we have the new to repeat this at least twice. For now, a simple fetch in the template is enough to prevent a useless sub query.